docs/development/source-code-directory-structure.md

```diff
@@ -1,15 +1,13 @@
-# Source code directory structure
+# Source Code Directory Structure
 
-## Overview
-
-The source code of Electron is separated into a few parts, and we are mostly
+The source code of Electron is separated into a few parts, mostly
 following Chromium on the separation conventions.
 
 You may need to become familiar with [Chromium's multi-process
 architecture](http://dev.chromium.org/developers/design-documents/multi-process-architecture)
 to understand the source code better.
 
-## Structure of source code
+## Structure of Source Code
 
 * **atom** - Source code of Electron.
```

Comment:
So this is nested 2 levels beyond what we can nest in the docs on electron.atom.io but that nesting is important to the meaning so I'm not sure what to do except maybe do it in this kind of style:

```text
_ dir
  |_dir
```